### DISCUSSION TITLE
Feature request: Define style for caret line in line numbers

### DISCUSSION THREAD

**Discussion by paaguti** [orbitalquark/textadept#343] [Ideas]
Hi,

I didn't see anything in QScintilla, so this might be a feature request to upstream. But would it be possible to create a style for the line number for the line holding the current. Something like:

![image](https://user-images.githubusercontent.com/3320432/213631510-f75fe32f-2cdd-497c-8b1f-2235d61b15d8.png)

Having this visual feedback would overcome the fact that selection(s) are hidden if you define the background for the current line.

Thx, /PA

**Comment by orbitalquark** [comment]
Yes, this would have to be an upstream request. I would like to have 
this feature too.
